packages/app/src/cli/services/deploy/theme-extension-config.test.ts

```diff
@@ -0,0 +1,37 @@
+import {themeExtensionConfig} from './theme-extension-config.js'
+import themeSpec from '../../models/extensions/theme-specifications/theme.js'
+import {ThemeExtensionInstance} from '../../models/extensions/theme.js'
+import {inTemporaryDirectory, writeFile, mkdir} from '@shopify/cli-kit/node/fs'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {describe, expect, test} from 'vitest'
+
+describe('themeExtensionConfig', () => {
+  test('throws an error if the request to return the url errors', async () => {
```

Comment:
Is this test title matching the test body?

Reply:
fixed, thanks!